docs/tutorials/01_introductory/viz_texture.py

```diff
@@ -0,0 +1,40 @@
+"""
+===============
+Sphere Texture
+===============
+In this tutorial, we will show how to create a sphere with a texture.
+"""
+
+import numpy as np
+from fury import window, actor, utils, primitive, io
+from fury.data import read_viz_textures, fetch_viz_textures
+
+##############################################################################
+# Create a a scene to start.
+
+scene = window.Scene()
+
+##############################################################################
+# Load an image (png, bmp, jpeg or jpg) using ``io.load_image``. In this
+# example, we will use ``read_viz_textures`` to access an image of the
+# Earth's surface from the fury github after using ''fetch_viz_textures()''
+# to download the available textures.
+
+fetch_viz_textures()
+filename = read_viz_textures("1_earth_8k.jpg")
+image = io.load_image(filename)
+
+##############################################################################
+# Next, use ``actor.texture_on_sphere`` to add a sphere with the texture from
+# your loaded image to the already existing scene.
+# To add a texture to your scene as visualized on a plane, use
+# ``actor.texture`` instead.
+
+scene.add(actor.texture_on_sphere(image))
+
+##############################################################################
+# Lastly, record the scene. If you want to manipulate your new sphere with a
+# texture, use ``window.show`` as commented out.
+
+# window.show(scene, size=(600, 600), reset_camera=False)
+window.record(scene, size=(900, 768), out_path="viz_texture.png")
```

Comment:
Wouldn't it be better if we have something like this:
```python
interactive = False
if interactive:
    window.show(...)
window.record(...)
```
Like the way we have for all the other tutorials?

Reply:
Thanks for pointing this out! I have changed the tutorial to follow your suggestion, making it consistent with the other tutorials.